packages/ts/react-crud/src/autoform-field.tsx

```diff
@@ -133,6 +140,16 @@ export type AutoFormFieldProps = CheckboxProps &
 export function AutoFormField(props: AutoFormFieldProps): JSX.Element | null {
   const { form, propertyInfo, options } = props;
   const label = options?.label ?? propertyInfo.humanReadableName;
+
+  const formPart = useFormPart(getPropertyModel(form, propertyInfo));
+  useEffect(() => {
+    if (options?.validators) {
+      options.validators.forEach((validator) => {
+        formPart.addValidator(validator);
+      });
+    }
+  }, [formPart, options]);
```

Comment:
This will add new validators with every single render. The only way to make this work is if developers would memoize (`useMemo`) the validators before passing them into the options, which no one will think of doing. I have a hard time seeing how the props approach could work here.

Reply:
An alternative might be to expose the `form` instance (from `useForm`) through a ref. Though I'm not sure if you can add validators to individual fields through that.